tests/x11/gnucash.pm

```diff
@@ -32,16 +33,9 @@ sub run {
         send_key 'alt-c';
         assert_screen('test-gnucash-tips-closed');
     }
-    send_key 'ctrl-q';    # Exit
-
-    # arbitrary limit
-    for (1 .. 7) {
-        assert_screen [qw(test-gnucash-1 gnucash gnucash-save-changes generic-desktop)];
-        last              if match_has_tag 'generic-desktop';
-        send_key 'alt-c'  if match_has_tag 'test-gnucash-1';
-        send_key 'alt-w'  if match_has_tag 'gnucash-save-changes';
-        send_key 'ctrl-q' if match_has_tag 'gnucash';
-    }
+    assert_and_click('gnucash-close-window');
+    assert_and_click('gnucash-close-without-saving-changes') if (is_leap('<=15.1'));
+    assert_screen('generic-desktop');
```

Comment:
This is already ensured by the post_run_hook, see the double check for generic-desktop http://10.160.66.74/tests/913#step/gnucash/22

Reply:
If this module is not able to close the window, I want it to fail, not making the next module fail.

Reply:
Yeah I think the post_run_hook will do exactly that but you can try to prove me wrong :)